### PR TITLE
[Android/Doc] Add guide for using NNAPI delegate to README

### LINKED ISSUE #2820: [Android/tflite] NNAPI fails to utilize available devices like dsp, gpu, or npu
When using NNAPI delegate, it should utilize hardware devices like dsp, gpu, hta (for qualcomm snapdragon). But it fails to use them, and use `nnapi-reference` which is CPU.

```
...
I/ExecutionPlan: ModelBuilder::findBestDeviceForEachOperation(CONV_2D) = 4 (nnapi-reference)
...
```

It should be like this (log of the tensorflow lite java example from the official repo):
```
...
I/ExecutionPlan: ModelBuilder::findBestDeviceForEachOperation(CONV_2D) = 0 (qti-default)
...
```

### PR DESCRIPTION
**Pull request by anyj0527**
- Add guide for using TensorFlow Lite NNAPI delegate to README.md

This resolves #2820 

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: api/android/README.md [hunk 1]
```diff
@@ -235,3 +235,43 @@ You can find the result in ```$NNSTREAMER_ROOT/android_lib```.
...
+
+If you want to use NNAPI delegate, you should provide the model file from internal storage. You may copy the model files into app-specific directory or cache directory. Or let your application access model files reside in the asset directory.
+
+A Java Example:
+
+```java
+...
+...
+
+String root = Environment.getExternalStorageDirectory().getAbsolutePath();
```
Comment (jaeyun-jung) [inline api/android/README.md:252]: Let's consider two methods (do not use instrument test method here)

File modelFromCacheDir(Context context) {
...
}

File modelFromFilesDir(Context context) {
}

Comment (anyj0527) [reply api/android/README.md]: That would be more clear. I updated it!

**Review by myungjoo — APPROVED** [review]
LGTM

File: api/android/README.md [hunk 2]
```diff
@@ -235,3 +235,44 @@ You can find the result in ```$NNSTREAMER_ROOT/android_lib```.
...
+
+```java
+/*
+ * Copy a model file in external storage into app-specific internal storage and return it
+ *
+ * @param context  The application context
+ * @param model    The File object of a model file in external storage
+ */
+File modelFromFilesDir(Context context, File model) {
+  File appSpecificFile = new File(context.getFilesDir(), "model_file_in_FilesDir.tflite");
```
Comment (jaeyun-jung) [inline api/android/README.md:256]: new File(context.getFilesDir(), model.getName()); maybe this is right one

Comment (anyj0527) [reply api/android/README.md]: Yeah that would be great! I updated this.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**